.pipelines/ci.yml

```diff
@@ -0,0 +1,75 @@
+# Azure DevOps Pipeline running CI
+trigger:
+  - master
+
+variables:
+  - template: vars.yml
+
+jobs:
+  - job: "Python_Unit_Tests"
+    pool:
+      vmImage: "ubuntu-latest"
+    strategy:
+      matrix:
+        Python27:
+          python.version: "2.7.17"
+        Python35:
+          python.version: "3.5.9"
+        Python36:
+          python.version: "3.6.10"
+        Python37:
+          python.version: "3.7.6"
+        Python38:
+          python.version: "3.8.1"
+
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "$(python.version)"
+      - template: ./templates/template-setup-golang-env.yml
+        parameters:
+          gobin: ${{ variables.GOBIN }}
+          gopath: ${{ variables.GOPATH }}
+          goroot: ${{ variables.GOROOT }}
+          modulePath: ${{ variables.modulePath }}
+      - script: |
+          set -x
+          pip install virtualenv
+          make test-python
+        displayName: "🧪Run Python Unit Tests : $(python.version)"
+        workingDirectory: "${{ variables.modulePath }}"
+
+  - job: "Golang_Unit_Tests"
+    pool:
+      vmImage: "ubuntu-latest"
+
+    steps:
+      - template: ./templates/template-setup-golang-env.yml
+        parameters:
+          gobin: ${{ variables.GOBIN }}
+          gopath: ${{ variables.GOPATH }}
+          goroot: ${{ variables.GOROOT }}
+          modulePath: ${{ variables.modulePath }}
+      - script: |
+          set -x
+          make test-go
+          [[ -z "$(git status -s)" ]]
+          bash -c 'go-junit-report < uts.txt > report.xml'
```

Comment:
why `bash -c` out of interest?

Reply:
Looks like the `<` and `>` are badly interpreted by the vsts agent. 
Figured this was working.